lib/store_model/enum.rb

```diff
@@ -46,7 +46,6 @@ def define_predicate_methods(name, mapping, options)
 
     def define_map_readers(name, mapping)
       define_method("#{name}_values") { mapping }
-      alias_method(ActiveSupport::Inflector.pluralize(name), "#{name}_values")
```

Comment:
Can we keep it but add `if name.singularize == name`? So the alias will only be added when name is singular

Reply:
@DmitryTsepelev I can definitely do that, but I wonder if that behavior will be surprising to users of this library. Specifically, the method would be a shortcut for the underlying mapping in some cases, but a reader for the actual value in others. I understand there might be a backwards-compatibility concern, but what do you think?

Reply:
Well, removing this right now will be backward incompatible as well 😅I agree that behavior that changes based on the _name_ of the attribute is a bit confusing